otherFiles/ImmutableExample.java

```diff
@@ -0,0 +1,21 @@
+import java.util.*;
+
+public class ImmutableExample {
```

Comment:
Not quite sure what this class does. 

You probably want to rename start() into demo() and demonstrate how ImmutableClass is really immutable (if you change it -> it's a different class)